features/org.wso2.developerstudio.eclipse.swtfunctionalframework.feature/feature.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="org.wso2.developerstudio.eclipse.swtfunctionalframework.feature"
+      label="Feature"
+      version="4.0.0.qualifier">
+
+   <description url="http://www.example.com/description">
+      [Enter Feature Description here.]
```

Comment:
1. Add a proper description
2. Include licence and copyright URL as below

   <copyright url="%copyrightURL">
      %copyright
   </copyright>

   <license url="%licenseURL">
      %license
   </license>